src/window/mouse_manager.rs

```diff
@@ -67,6 +67,29 @@ struct TouchTrace {
     left_deadzone_once: bool,
 }
 
+struct ScrolledPixels {
+    x: f32,
+    y: f32,
+}
+
+impl ScrolledPixels {
+    fn new() -> ScrolledPixels {
+        ScrolledPixels { x: 0., y: 0. }
+    }
+
+    fn reset(&mut self) {
+        self.x = 0.;
+        self.y = 0.;
```

Comment:
```suggestion
        *self = Self::new();
```

doesn't require updating another place for theoretical future fields, defaults or type changes.